projects/cdk/utils/svg/svg-linear-gradient-processor.ts

```diff
@@ -40,9 +40,12 @@ function makeRandomSalt(): number {
 }
 
 function extractLinearGradientIdsFromSvg(svg: string): string[] {
```

Comment:
Open this example from your branch:
https://taiga-ui.dev/icons/SVG_Tips#inline

It is broken now
<img width="712" alt="Screenshot 2023-03-30 at 16 07 41" src="https://user-images.githubusercontent.com/35179038/228845682-3dd0ab94-c43e-4060-ab41-a0ea6fa99535.png">

Reply:
It's fixed, I added a Unit Test for this SVG too. It's a edgy case where the ids were the same as the html tag.